Makefile

```diff
@@ -8,7 +8,7 @@ build: test
 
 .PHONY: image
 image: build
-	docker build -t "quay.io/numaio/numaflow-source/nats-source-go:v0.5.0" --target nats-source .
+	docker build -t "quay.io/numaio/numaflow-source/nats-source:v0.5.10" --target nats-source .
```

Comment:
why change from `nats-source-go` to `nats-source`? And why v0.5.10? I thought we were going to do something like v0.1.0

Reply:
changed back to `nats-source-go` . In terms of version. I don't think it's 1.0.0 yet. Decide to keep it as 0.5.x for now.